dockerfile/avalanche/Dockerfile

```diff
@@ -139,7 +133,9 @@ RUN for b in \
   less \
   ls \
   md5sum \
+  mv \
```

Comment:
We can add `dirname` to the util bins here for remaining Dockerfiles, might as well since it is available within the minimal busybox.